### PR TITLE
Fixes for Julia 0.6

### PR DESCRIPTION
**Pull request by andreasnoack**

### CODE REVIEW

**Comment by Evizero** [comment]
Thanks for the effort!

**Comment by andreasnoack** [comment]
Thanks for the package.

File: src/graphics/bargraphics.jl [hunk 1]
```diff
@@ -1,3 +1,5 @@
+if VERSION < v"0.6-"
```
Comment (tkelman) [inline src/graphics/bargraphics.jl]: please be more specific with VERSION cutoffs like this, `where` isn't going to work for all 0.6-dev versions

Comment (Evizero) [inline src/graphics/bargraphics.jl]: what would be a way to find a reasonable lower bound? any pointer where to start looking. Is it related to the merge of https://github.com/JuliaLang/julia/pull/18457 ?

Comment (tkelman) [inline src/graphics/bargraphics.jl]: yes, check out the merge commit there and run contrib/commit-name.sh

**Pull request merged**